payments/models.py

```diff
@@ -432,10 +432,15 @@ def save(self, *args, **kwargs):
                 if isinstance(self.lease, WinterStorageLease) and isinstance(
                     self.product, WinterStorageProduct
                 ):
-                    place_sqm = (
-                        self.lease.place.place_type.width
-                        * self.lease.place.place_type.length
-                    )
+                    if self.lease.place:
```

Comment:
Could we create a test case for this to `test_payments_models.py` ?

Reply:
Good idea!